packages/grid/x-data-grid-premium/src/components/GridColumnMenuAggregationItem.tsx

```diff
@@ -76,29 +83,30 @@ function GridColumnMenuAggregationItem(props: GridColumnMenuItemProps) {
       <ListItemText>
         <FormControl size="small" fullWidth sx={{ minWidth: 150 }}>
           <InputLabel id={`${id}-label`}>{label}</InputLabel>
-          <Select
+          <rootProps.slots.baseSelect
+            aria-labelledby={`${id}-label`}
             labelId={`${id}-label`}
-            id={`${id}-input`}
             value={selectedAggregationRule}
             label={label}
+            native
```

Comment:
I think we can avoid using `native` here and fix the column menu shift with `mui-fixed` class - see https://mui.com/material-ui/getting-started/faq/#why-do-the-fixed-positioned-elements-move-when-a-modal-is-opened

```diff
diff --git a/packages/grid/x-data-grid/src/components/menu/columnMenu/GridColumnHeaderMenu.tsx b/packages/grid/x-data-grid/src/components/menu/columnMenu/GridColumnHeaderMenu.tsx
index 5c067a68b..a7027d560 100644
--- a/packages/grid/x-data-grid/src/components/menu/columnMenu/GridColumnHeaderMenu.tsx
+++ b/packages/grid/x-data-grid/src/components/menu/columnMenu/GridColumnHeaderMenu.tsx
@@ -51,6 +51,7 @@ function GridColumnHeaderMenu({
       target={target}
       onClickAway={hideMenu}
       onExited={onExited}
+      className="mui-fixed"
     >
       <ContentComponent
         colDef={colDef}
```

Reply:
I can't reproduce this one

Reply:
Correct, it's reproducible on Firefox (Mac OS), may be it's a very corner case, so neglect it?